scripts/jenkins-mender-convert-build.sh

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+
+LINARO_COMPILER=gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf
+COMPRESSED_COMPILER=${LINARO_COMPILER}.tar.xz
+LINARO_BINARY_PATH=${LINARO_COMPILER}/bin
+
+DEBIAN_IMAGE=bone-debian-9.4-iot-armhf-2018-06-17-4gb.img
+COMPRESSED_DEBIAN=${DEBIAN_IMAGE}.xz
+CONVERTED_DEBIAN_IMAGE=mender_debian_converted.sdimg
+
+RASPBIAN_FILE_NAME=2018-06-27-raspbian-stretch-lite
+RASPBIAN_IMAGE=${RASPBIAN_FILE_NAME}.img
+COMPRESSED_RASPBIAN=${RASPBIAN_FILE_NAME}.zip
+CONVERTED_RASPBIAN_IMAGE=mender_raspbian_converted.sdimg
+
+QEMUX86_64_BOARD_NAME=qemux86-64-ovmf-grub
+QEMUX86_64_RAW_DISK_IMAGE=
+CONVERTED_QEMUX86_64_IMAGE=mender_qemux86_64_converted.sdimg
+
+export GOPATH="$WORKSPACE/go"
+
+host=$(uname -m)
+
+get_arm_compiler() {
+    wget -nc -q https://releases.linaro.org/components/toolchain/binaries/6.3-2017.05/arm-linux-gnueabihf/gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf.tar.xz
+    tar -xJf ${COMPRESSED_COMPILER}
+}
+
+get_debian() {
+    wget -nc -q http://debian.beagleboard.org/images/bone-debian-9.4-iot-armhf-2018-06-17-4gb.img.xz
+    xz -d -k ${COMPRESSED_DEBIAN}
+}
+
+get_raspbian() {
+    wget -nc -q https://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2018-06-29/2018-06-27-raspbian-stretch-lite.zip
+    unzip -q ${COMPRESSED_RASPBIAN}
+}
+
+get_pytest_files() {
+    wget -nc -q https://raw.githubusercontent.com/mendersoftware/meta-mender/master/tests/acceptance/pytest.ini -P $WORKSPACE/mender-image-tests
+    wget -nc -q https://raw.githubusercontent.com/mendersoftware/meta-mender/master/tests/acceptance/common.py -P $WORKSPACE/mender-image-tests
+    wget -nc -q https://raw.githubusercontent.com/mendersoftware/meta-mender/master/tests/acceptance/conftest.py -P $WORKSPACE/mender-image-tests
+    wget -nc -q https://raw.githubusercontent.com/mendersoftware/meta-mender/master/tests/acceptance/fixtures.py -P $WORKSPACE/mender-image-tests
+}
+
+get_mender_artifact() {
+    wget -nc -q https://d1b0l86ne08fsf.cloudfront.net/mender-artifact/2.3.0/mender-artifact
+}
+
+build_mender_client() {
+    go get github.com/mendersoftware/mender
+    cd $GOPATH/src/github.com/mendersoftware/mender
+    git checkout 1.5.0
+    env CGO_ENABLED=1 \
+        CC=arm-linux-gnueabihf-gcc \
+        GOOS=linux \
+        GOARCH=arm make build
+    arm-linux-gnueabihf-strip mender
+    cp mender $GOPATH/bin/mender-arm
+    make build
+    strip mender
+    cp mender $GOPATH/bin/mender-$host
+    cd -
+}
+
+build_mender_artifact() {
```

Comment:
Is it needed at all? If you have it already pre-build binary downloaded in `get_mender_artifact()` method?